yang/frr-vrf.yang

```diff
@@ -0,0 +1,50 @@
+module frr-vrf {
+  yang-version 1.1;
```

Comment:
As part of https://github.com/FRRouting/frr/pull/5492 it is defined yang/frr-vrf.yang as it was missing. Also it seems you have raised PR against master but we are using nb_conversions branch to commit all NB changes for now.

Reply:
Created a new [PR-5501](https://github.com/FRRouting/frr/pull/5501) against nb_conversions brnach.